### PR TITLE
EWL-4281 - SG | Reconfigure "Topic Related Articles"

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

Please do not submit a Pull Request without a relevant ticket! If you are creating a new pattern or feature, create an issue describing the need for this feature Github **first** and then link to it below.

**Jira Ticket**

- [EWL-4281: Reconfigure "Topic Related Articles"](https://issues.ama-assn.org/browse/EWL-4281)

## Description

Adds pseudo pattern for related topics to show only title. 
Uses pseudo pattern to set classes
Moves topic label outside of child and in parent div

## To Test

- [x] Navigate to http://localhost:3000/?p=templates-topic
- [x] Navigate to http://localhost:3000/?p=templates-topic-title-only 
- [ ] Confirm that the look and feel is not broken and title only shows when wanted

## Relevant Screenshots/GIFs

screen shot 2017-11-22 at 9 07 31 am

## Remaining Tasks

Remaining tasks?

## Additional Notes

Anything more to add?

### CODE REVIEW

File: styleguide/source/_patterns/03-templates/04-topic/_topic.scss [hunk 1]
```diff
@@ -23,7 +23,11 @@
 .topic .grid-region {
 
   .grid-region_content:not(:first-child){
-    @include gutter($padding-top-full...);
+    @include gutter($padding-bottom-full...);
```
Comment (froboy) [inline styleguide/source/_patterns/03-templates/04-topic/_topic.scss]: pattern_lab_-_templates-topic
This breaks the spacing between the first and second blocks. If you're switching to bottom gutters then maybe the selector should be `:not(:last-child)`?

Comment (designerneil) [reply styleguide/source/_patterns/03-templates/04-topic/_topic.scss]: yeah. probably. will change

**Comment by froboy** [comment]
pattern_lab_-_templates-topic

@designerneil sorry, not quite. It looks like `grid-region_content` is on two levels of element, so now there's extra space between the element. Might want to reconsider how this spacing is working.

Also, in other news, the Title only psudopattern looks good on its own, but on http://localhost:3000/?p=templates-topic-title-only all of the articles in all of the regions are getting title-only. this should only effect the left sidebar block. 
pattern_lab_-_templates-topic-title-only

**Comment by designerneil** [comment]
@froboy thanks for your review. I made the changes you requested.

File: styleguide/source/_patterns/03-templates/04-topic/_topic.scss [hunk 2]
```diff
@@ -42,6 +38,11 @@
       grid-area: 1 / 1 / 4 / 1;
       padding-left: 14px;
     }
+
+    .grid-region_content.topic_topic-articles > div {
+      padding-top: 0;
+      @include gutter($padding-bottom-full...);
```
Comment (froboy) [inline styleguide/source/_patterns/03-templates/04-topic/_topic.scss]: @designerneil almost there... if you reorder the blocks on the page (put the promo after the topic-articles block in the left sidebar, for instance), you'll see the space below the last item is too large now. Try wrapping this line in `:not(:last-child)`:

```
    .grid-region_content.topic_topic-articles > div {
      padding-top: 0;
      &:not(:last-child) {
        @include gutter($padding-bottom-full...);
      }
    }
```

**Comment by froboy** [comment]
@designerneil awesome. Thanks a ton. I'm going to let @chelsiejohnston take one final pass at this (so that a real FED sees it) but otherwise I approve.

File: styleguide/source/_patterns/03-templates/04-topic/_topic.scss [hunk 3]
```diff
@@ -38,6 +38,13 @@
       grid-area: 1 / 1 / 4 / 1;
       padding-left: 14px;
     }
+
+    .grid-region_content.topic_topic-articles > div {
```
Comment (froboy) [inline styleguide/source/_patterns/03-templates/04-topic/_topic.scss]: Ugh... so... Drupal is rendering these as `<article>` and the style guide is rendering them as `<div>`. I'm not sure if fixing one or the other is in scope here, but we need to at least change this so it's not calling divs. Any reason it couldn't be `> .topic_article-preview`?

**Comment by froboy** [comment]
horizontal lines: i think we should leave this out since the "no title" version is meant to mimic the old related content block that doesn't have separators. I think the spacing works well enough.
pattern_lab_-_templates-topic

spacing issue: my [comment above](https://github.com/AmericanMedicalAssociation/AMA-style-guide/pull/266#discussion_r153265668) will resolve this.

**Comment by designerneil** [comment]
@chelsiejohnston the fonts have been updated per your suggestion

**Comment by chelsiejohnston** [comment]
Handful of things:

- Please rename pseudopattern to something with a more accurate title instead of "Topic Title Only" (something like "Topic with Sidebar Block Title Only"?)

- The font size for the sidebar block still looks incorrect. It should be 21px on desktop like the existing Related Content block. It looks correct on mobile and tablet though.

- Testing in d8, can we make the first article in the block have `padding-top: 14px;` rather than the full 28px? The spacing looks too much between the label and the first article: 
thing-1

Other than these things I think this is good to go

**Pull request merged**